johnsonandjohnson/Bodiless-JS
# perf(components,layouts): Improve performance of heavy component selectors

## Changes
- Do not apply flow container or chameleon designs to all available components until the component selector is opened.
- 
## Test Instructions
- Regression Testing (edit & static) of existing chameleons and flow containers, including
  - /flow-container page
   - /chameleon page
   - menus
   - link toggle
- new cs-perf page (verify that page loads in a reasonable time).
- 
## Related Issues
Closes #1236
